docs/chapters/derivatives/content.zh-CN.md

```diff
@@ -0,0 +1,149 @@
+# 导数
+
+利用贝塞尔函数的导数可以对贝塞尔曲线做一些有用的事，而贝塞尔函数较为有趣的一个性质是其导数也为贝塞尔函数。其实贝塞尔函数的求导相对而言比较直接，只是需要一点数学运算。
+
+首先观察贝塞尔函数的求导法则，即：
+
+\[
+  \textit{Bézier}'(n,t) = n \cdot \sum_{i=0}^{n-1} (b_{i+1}-b_i) \cdot \textit{Bézier}(n-1,t)_i
+\]
+
+上式可改写为（注意式中的*b*即权重*w*，且*n*乘以一个和式等于每个求和项乘以*n*再求和）：
+
+\[
+  \textit{Bézier}'(n,t) = \sum_{i=0}^{n-1} \textit{Bézier}(n-1,t)_i \cdot n \cdot (w_{i+1}-w_i)
+\]
+
+直白地说，*n*次贝塞尔函数的导数为*n*-1次贝塞尔函数，少了一项，而且新的权重*w'*<sub>0</sub>、……、*w'*<sub>*n*-1</sub>可用旧的权重通过*n*(*w*<sub>*i*+1</sub> - *w*<sub>*i*</sub>)求得。对于带四个权重的三次函数，其导数的三个新权重为：*w'*<sub>0</sub> = 3(*w*<sub>1</sub>-*w*<sub>0</sub>)，*w'*<sub>1</sub> = 3(*w*<sub>2</sub>-*w*<sub>1</sub>)和*w'*<sub>2</sub> = 3(*w*<sub>3</sub>-*w*<sub>2</sub>)。
+
+<div class="note">
+
+### “慢着，为什么这是对的？”
+
+虽然有时候有人告诉说“这是导数”就行，但还是可能想一探究竟。既然如此，就来看看这个导数的证明。首先，因为权重不影响完整的贝塞尔函数的求导，所以求导只涉及多项式基函数的导数。基函数的导数为：
+
+\[
+  \frac{\mathrm{d}}{\mathrm{d}t} B_{n,k}(t) = \frac{\mathrm{d}}{\mathrm{d}t} \left( \mathrm{C}_n^k t^k (1-t)^{n-k} \right)
+\]
+
+使用[乘积求导法则](https://mathworld.wolfram.com/ProductRule.html)和[复合函数求导法则](https://mathworld.wolfram.com/ChainRule.html)得到：
```

Comment:
We should be able to use whatever chinese articles on wikipedia exist for the chain and product rule here, rather than having to send folks to a English-only website.

Reply:
By "inaccessibility" I mean that Wikipedia is blocked in mainland China. There're a couple of solutions here with different pros and cons:

1. Use Wikipedia anyway
Pro(s): The translation adheres to the English version; linked pages are available in Chinese.
Con(s): In mainland China, links can only be accessed by IT professionals using VPNs.
2. Use Mathworld/Planetmath
Pro(s): Almost all math-related Wikipedia topics are available; links are not blocked in mainland China.
Con(s): Linked pages are only available in English.
3. Use 万维百科, a website running by community volunteers that fetches contents from Simplified Chinese version of Wikipedia
Pro(s): Links are not blocked in mainland China; linked pages are available in Chinese.
Con(s): A siginificant proportion of Wikipedia pages are missing, including some referenced here in the Primer.

The reason why I chose Approach 2: I'd like to keep both adherence and readership as high as possible, and Approach 2 is the best tradeoff in this regard. As for language availability: auto-translation addons for browsers are so popular that it's no longer a blocking issue in the 20's.

However, I'll abide by what the original author deems most appropriate :) Since the decision affects all chapters, I'll wait and then make fixes for all the translated or review-pending chapters.

Reply:
in that case, let's go with #2

Reply:
However, I will be merging these with the wikipedia links for now, so that the content's in, and then we can update the links in a separate issue